spec/models/edo_oracle/career_spec.rb

```diff
@@ -2,18 +2,72 @@
+    context 'user id 790833' do
+      let(:uid) { 790833 }
+      it 'returns the expected result' do
+        expect(subject).to be
+        expect(subject.count).to be 2
+
+        expect(subject[0].count).to eq 10
+        expect(subject[0]).to have_keys(%w(acad_career program_status total_cumulative_units total_cumulative_law_units))
+        expect(subject[0]).to have_keys(%w(total_transfer_units transfer_units_adjustment ap_test_units ib_test_units alevel_test_units total_transfer_units_law))
+
+        expect(subject[1].count).to eq 10
+        expect(subject[1]).to have_keys(%w(acad_career program_status total_cumulative_units total_cumulative_law_units))
+        expect(subject[1]).to have_keys(%w(total_transfer_units transfer_units_adjustment ap_test_units ib_test_units alevel_test_units total_transfer_units_law))
+      end
+    end
+
+    context 'user id 300216' do
+      let(:uid) { 300216 }
+      it 'returns the expected result' do
+        expect(subject).to be
+        expect(subject.count).to be 3
+
+        expect(subject[0].count).to eq 10
+        expect(subject[0]).to have_keys(%w(acad_career program_status total_cumulative_units total_cumulative_law_units))
+        expect(subject[0]).to have_keys(%w(total_transfer_units transfer_units_adjustment ap_test_units ib_test_units alevel_test_units total_transfer_units_law))
+
+        expect(subject[1].count).to eq 10
+        expect(subject[1]).to have_keys(%w(acad_career program_status total_cumulative_units total_cumulative_law_units))
+        expect(subject[1]).to have_keys(%w(total_transfer_units transfer_units_adjustment ap_test_units ib_test_units alevel_test_units total_transfer_units_law))
 
-      expect(subject[1].count).to eq 10
-      expect(subject[1]).to have_keys(%w(acad_career program_status total_cumulative_units total_cumulative_law_units))
-      expect(subject[1]).to have_keys(%w(total_transfer_units transfer_units_adjustment ap_test_units ib_test_units alevel_test_units total_transfer_units_law))
+        expect(subject[2].count).to eq 10
+        expect(subject[2]).to have_keys(%w(acad_career program_status total_cumulative_units total_cumulative_law_units))
+        expect(subject[2]).to have_keys(%w(total_transfer_units transfer_units_adjustment ap_test_units ib_test_units alevel_test_units total_transfer_units_law))
+      end
+    end
+  end
+
+  describe '#get_cumulative_units' do
+    subject { described_class.new({user_id: uid}).get_cumulative_units }
+    context 'user id 790833' do
+      let(:uid) { 790833 }
+      it 'returns the expected result' do
+        expect(subject).to be
+        expect(subject.count).to be 1
+        expect(subject[0]).to have_keys(%w(acad_career program_status total_cumulative_units total_cumulative_law_units))
+        expect(subject[0]['total_cumulative_units']).to eq 2.0
+        expect(subject[0]['total_cumulative_law_units']).to be_nil
+      end
+    end
+
+    context 'user id 300216' do
```

Comment:
@rishab231 - Is it possible for us to use a description in the 'context' description for what the student UID 300216 represents? Is this supposed to represent a student with a certain state? Like an undergrad with units, or something like that?